rest-api/how-to-deploy-rest.sh

```diff
@@ -0,0 +1,194 @@
+## IMPORTANT: this file and accompanying assets are the source for snippets in https://docs.microsoft.com/azure/machine-learning! 
```

Comment:
if moving this file to this directory, need to delete the other one in /cli and update the docs